azurerm/internal/services/appplatform/spring_cloud_service_resource.go

```diff
@@ -133,8 +144,38 @@ func resourceArmSpringCloudService() *schema.Resource {
 				},
 			},
 
+			"trace": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"enabled": {
+							Type:     schema.TypeBool,
+							Required: true,
+						},
+
+						"app_insight_instrumentation_key": {
```

Comment:
we can shorten this to `instrumentation_key` - since no other Azure Service uses this terminology